examples/drivers/windows/otLwf/radio.c

```diff
@@ -814,6 +814,14 @@ void otPlatRadioSetDefaultTxPower(_In_ otInstance *otCtx, int8_t aPower)
     }
 }
 
+int8_t otPlatRadioGetReceiveSensitivity(_In_ otInstance *otCtx)
+{
+    NT_ASSERT(otCtx);
+    PMS_FILTER pFilter = otCtxToFilter(otCtx);
+    UNREFERENCED_PARAMETER(pFilter);
+    return -100;
```

Comment:
This should be exposed via a Spinel/NCP property that wraps a call to a LinkRaw API that calls this on the device.